spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('Intervention component', function () {
+  var FIXTURE
+
+  function loadInterventionComponent () {
+    var element = document.querySelector('[data-module="gem-intervention"]')
+    new GOVUK.Modules.Intervention(element).init()
+  }
+
+  beforeEach(function () {
+    spyOn(GOVUK.analytics, 'trackEvent')
+    spyOn(GOVUK.Modules, 'GemTrackClick').and.callFake(function () { this.start = function () {} })
+
+    FIXTURE = `
```

Comment:
For readability of the `beforeEach` function, I'd move this out of here to the top of the file.